### PR TITLE
Blog post about the recent changes in the PO group role titles and profiles

### PR DESCRIPTION
**Pull request by alixgeorge**
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by sparkymarky79** [comment]
I love this, really does sum up what has happened, what is happening and the iterations we go through to get the process to work for the company and team.

**Comment by carolinewoodhams** [comment]
I won't take a formal review as I've already proof read - but would you mind changing the categories and tags? (Trying to tidy these up going forwards & make them work harder for us!)

Categories - Product Owner, Pod Lead
Tags - Managing people, scrum, roles

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+comments: "true"
+layout: post
+published: false
+title: The group formerly known as POs
+author: Alix George
+categories: "Managing people, Product Owner, Scrum, Roles"
+---
+## Changing role titles and profiles
+
+I was asked recently why we've made some recent changes to some of the role titles and profiles for some of the group formerly known as the Product Owners. To best answer, I decided to write a blog post about it.
```
Comment (notABluesSinger) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: Could you drop the repetition of "recent"?

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The main reason behind the changes was to set them up for success. That can sound a bit cliche but it's something I say often in my role - management is about setting people up for success. If someone is in the right role with the right expectations, it's great for them, their development and growth, and great for the business.
```
Comment (notABluesSinger) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: I wonder if you could add a little more clarity here, perhaps
```
The main reason behind the changes was to set each product owner up for success
```
?

Comment (notABluesSinger) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: Should it be "clichéd"?

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+So were the former role titles _wrong_? 
+
+Not exactly. 
+
+When the Web team started fully following scrum practices, the leads of each team (or pods as we call them at Holiday Extras) became Product Owners as per the framework. As a collective, we were known as the PO group.
+
+We had a shared role profile that outlined what 'great' looks like for a PO, which listed qualities we need, responsibilities in the role and the approach required to deliver against those responsibilities successfully. 
+
+I became a Product Owner in January 2015 and then Lead Product Owner in October the same year - when I began managing the other PO's, it was clear that what was great for one PO wasn't the same as for another.
```
Comment (notABluesSinger) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: I think you can drop the "as";
```
wasn't the same for another
```

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+From my side, I wanted to make sure that I was managing these individuals properly too. I wanted to make sure that people felt clear and confident in what was expected of them and that I could support them in the right way to do those things.
+
+Speaking to the individuals was an important part of this adjustment - and as of this week, we've nearly finished firming up their role profiles. 
+
+We now have a UK Delivery Manager, an EU Delivery Manager and a Technical Product Manager and to reflect the mix of roles, the PO group is now known as the Pod Leads. I see this as being the start of identifying what 'great' looks like for each Lead for each pod.
+
+I'm still working on my own role title - Lead Pod Lead, Pod Lead Lead, and PodLeady McLeadFace have been the suggestions (and rejections... not a poll, guys...) so far.
+
+This is one of the things I love about Holiday Extras - you try something, you evolve it, try something different until you find what works best (and even then, you keep trying!).
```
Comment (notABluesSinger) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]:
```
and even then, you still keep trying
```
?

**Comment by notABluesSinger** [comment]
Taking Slot 2️⃣.
Really good read and a great example of the agile nature of ur business, just a few small, pedantic tweaks from me 😄.

**Comment by alixgeorge** [comment]
Thanks @notABluesSinger - github noob so I've addressed your comments. :)

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md (hunk 4 above)
Comment (alixgeorge) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: Changed from 'best' to 'better'.

File: _posts/2016-05-18-the-group-formerly-known-as-pos.md (hunk 2 above)
Comment (alixgeorge) [inline _posts/2016-05-18-the-group-formerly-known-as-pos.md]: Thanks for the feedback - re. the clarity around setting people up for success, I set it up by saying people need to be in the right role with the right expectations and then the rest of the blog post goes into the issues that arose from them not being accurate so this is deliberate. :)

**Comment by notABluesSinger** [comment]
Nice work @alixgeorge :shipit:

**Comment by lyndseystapley** [comment]
This is really good Alix, gives a really good explanation of the changes and why they were needed :)

**Pull request merged**